src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -1647,6 +1647,9 @@ void Unit::CalcAbsorbResist(Unit* victim, SpellSchoolMask schoolMask, DamageEffe
 
     RoundToInterval(auraAbsorbMod, 0.0f, 100.0f);
 
+    auto absorb_ignoring_damage = CalculatePct(dmgInfo.GetDamage(), auraAbsorbMod);
```

Comment:
Thanks for the suggestion, I agree. Does it look OK if I name the variable `absorbIgnoringDamage` like this?
```cpp
    uint32 absorbIgnoringDamage = CalculatePct(dmgInfo.GetDamage(), auraAbsorbMod);
    dmgInfo.ModifyDamage(-absorbIgnoringDamage);
```
[...]
```cpp
    dmgInfo.ModifyDamage(absorbIgnoringDamage);